protocol/contracts/farm/facets/SeasonFacet/Weather.sol

```diff
@@ -64,15 +64,16 @@ contract Weather is Sun {
         );
 
         // Calculate Delta Soil Demand
-        uint256 dsoil = s.w.startSoil.sub(endSoil);
-
+        uint256 dsoil = s.f.beanSown;
+        s.f.beanSown = 0;
+    
         Decimal.D256 memory deltaPodDemand;
 
         // If Sow'd all Soil
         if (s.w.nextSowTime < type(uint32).max) {
             if (
                 s.w.lastSowTime == type(uint32).max || // Didn't Sow all last Season
-                s.w.nextSowTime < 300 || // Sow'd all instantly this Season
+                s.w.nextSowTime < 600 || // Sow'd all instantly this Season
```

Comment:
Move this to a constant.

Reply:
any ideas for names? I put `SOWTIMEDEMAND` for now, but think theres a better name out there.